src/components/chat-message.tsx

```diff
@@ -54,37 +80,32 @@ export const ChatMessage = ( {
 			<div
 				id={ id }
 				role="group"
+				data-testid="chat-message"
 				aria-labelledby={ id }
 				className={ cx(
+					'inline-block p-3 rounded border border-gray-300 overflow-x-auto select-text',
 					'inline-block p-3 rounded border overflow-x-auto select-text',
```

Comment:
@kozer Is there a reason we are adding this line twice - once with `border-gray-300 ` and the second tine without it but with all other identical classes?

Reply:
Good catch Kat! No, just leftover from while no was working on it. Feel free to clean it up! Thanks for looking at this one!